packages/python/plotly/plotly/express/imshow_utils.py

```diff
@@ -0,0 +1,248 @@
+"""Vendored code from scikit-image in order to limit the number of dependencies
+Extracted from scikit-image/skimage/exposure/exposure.py
+"""
```

Comment:
it's the exact scikit-image code for the `rescale_intensity` function, plus other bits from scikit-image moved here so that we use a single file.